include/CLI/Option.hpp

```diff
@@ -51,7 +51,7 @@ template <typename CRTP> class OptionBase {
 
   protected:
     /// The group membership
-    std::string group_ = std::string("Options");
+    std::string group_ = std::string("OPTIONS");
```

Comment:
Is changing this to ALL CAPS considered a breaking change @henryiii

Reply:
While not being asked, just my two cents: Where ever a hard check for the not all caps string was present, I adjusted the string to all caps when necessary. I scrolled through the code and until now did not find any case where the change could break something. Only `ConfigBase::to_config` directly had dependencies on the string to generate the description text in config files. I changed that, but that does not break config files, since it's just affecting the comments inside the files. It's not a guarantee but I wouldn't expect any incompatibilities with existing Apps using CLI11.

Reply:
Sounds fine to me.